all_repos/config.py

```diff
@@ -52,6 +52,8 @@ def load_config(filename: str) -> Config:
 
     output_dir = os.path.join(filename, '..', contents['output_dir'])
     output_dir = os.path.normpath(output_dir)
+    if os.path.abspath(output_dir) == os.getcwd():
+        raise SystemExit('setting output_dir to the current directory is not supported')
```

Comment:
better I think would be to check up front if the working directory structure looks like what we expect -- either it has the `all-repos` files and some folders and nothing else, or it's an error

Reply:
also the better way to write the check you've written is `os.path.samefile(output_dir, '.')`

Reply:
Rewrote the logic to do as you asked